docs_outdoornav_user_manual/features/watchdogs.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Watchdogs
+sidebar_label: Watchdogs
+sidebar_position: 7
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+As of version 0.14.0, the OutdoorNav software contains the ability for users to set
+up Points of Interest (POI) on their map. Each UGV will come pre-loaded with a set of
+watchdogs that are required for the overall safety of the system. Users are then able
+to add/remove/modify watchdogs as they require for their specific application and
+safety requirements. This will have a lot to do with the operational environment
+and their network setup. A history of previous watchdogs that have triggered is also
+to the user for more precise debugging of when issues may have occured.
+
+## Watchdog Types
+
+Currently there a a set of watchdogs that have been developed. Users are unable to write
+their own watchdogs at the present time.
+
+| Watchdog Type                    | Description                                            |
+|----------------------------------|----------------------------------------|
+| **Communication Watchdog**       | A watchdog used to monitor device communication with the PC. It will monitor and trigger if there is loss of communication on devices that are connected via the ethernet (ip) or CAN bus (can) communication protocols. |
+| **Heartbeat Watchdog**           | A watchdog used to monitor the heartbeat topic of a device. This differs from the Communication, Node Status and Topic Data watchdogs because there may be devices that are deemed to be alive and communicating by the Node Status and Communication watchdogs and but may not be publishing data on their published topic. FOr example some GNSS units will not publish to their if topic unless they are in a clear open environment, however the heartbeat topic will still be publishing since the node is alive. |
```

Comment:
by the Node Status and Communication watchdogs (**and**) but may not be publishing (remove the and)

Reply:
For example some GNSS units (capital O)

Reply:
will not publish to their (**if**) topic unless (remove the if)